src/data-fetchers/vcf/utils.ts

```diff
@@ -48,8 +48,8 @@ export function recordToTile(vcfRecord: VcfRecord, chrPos: number, prevPos?: num
     }
 
     // Additionally inferred values
-    const DISTPREV = !prevPos ? null : vcfRecord.POS - prevPos;
-    const DISTPREVLOGE = !prevPos ? null : Math.log(vcfRecord.POS - prevPos);
+    const DISTPREV = !prevPos ? null : POS - prevPos;
```

Comment:
`prevPos` was the absolute genomic position, but vcfRecord.POS is the relative one